container/docker/Dockerfile.user_conf

```diff
@@ -0,0 +1,3 @@
+FROM verizonmedia/yavin_base:latest
```

Comment:
Can we name these
- Dockerfile.base
- Dockerfile.demo-config
- Dockerfile.custom-config